packages/grafana-data/src/transformations/standardTransformersRegistry.ts

```diff
@@ -28,8 +28,27 @@ export interface TransformerRegistryItem<TOptions> extends RegistryItem {
    * React component used as UI for the transformer
    */
   editor: React.ComponentType<TransformerUIProps<TOptions>>;
+
+  /**
+   * Set of categories associated with the transformer
+   */
+  categories?: Set<TransformerCategory>;
+
+  /**
+   * Image path for the transformer
+   */
+  image?: { dark: string; light: string };
 }
 
+export type TransformerCategory =
```

Comment:
maybe an enum is better here?

Reply:
I tried with an enum but then I couldnt export it and use it as a value from other files. Maybe there is somethign missing in our ts config, or maybe I am doing somethign wrong here